src/interface_adapter/dashboard/DashboardViewModel.java

```diff
@@ -0,0 +1,34 @@
+package interface_adapter.dashboard;
+
+import interface_adapter.ViewModel;
+
+import java.beans.PropertyChangeListener;
+import java.beans.PropertyChangeSupport;
+
+public class DashboardViewModel extends ViewModel {
+    public final String TITLE_LABEL = "Project Dashboard";
+
```

Comment:
I need you to have a viewName attribute and also a getViewName() method for the viewmanager to change active view to dashboard. Those are defined in ViewModel.java.

Reply:
Great point! I have added these in DashboardViewModel and have pushed these changes.